server/pages/index.js

```diff
@@ -337,10 +337,10 @@ module.exports = function() {
 		} );
 	}
 
-	app.get( '/theme', ( req, res ) => res.redirect( '/design' ) );
-	// Interim redirect before we make `/themes` the canonical showcase URL.
-	app.get( [ '/themes', '/themes/*' ], ( req, res ) => {
-		res.redirect( '/design' + req.originalUrl.slice( '/themes'.length ) );
+	app.get( '/theme', ( req, res ) => res.redirect( '/themes' ) );
+	// Interim redirect before this is handled on server route config level
```

Comment:
Good question -- I'm not sure. I read Barry's comment as pertaining to finer-grained redirects, while this would be about bulk-redirecting `/design/*` -> `/themes/*`. It shouldn't matter too much -- we can drop it if they do it for us, or keep it and change the comment, I'd say.